pkg/pool-hooks/test/foundry/ExitFeeHookExample.t.sol

```diff
@@ -39,6 +42,8 @@ contract ExitFeeHookExampleTest is BaseVaultTest {
         super.setUp();
 
         (daiIdx, usdcIdx) = getSortedIndexes(address(dai), address(usdc));
+
+        poolFactoryMock = PoolFactoryMock(address(vault.getPoolFactoryMock()));
```

Comment:
Since this test only uses the PoolFactoryMock, maybe it should use the base vault test factory?

Reply:
Good catch! Thanks!

Reply:
unfortunately we need this because this contract is used in other tests